plugins/catalog-graphql/README.md

```diff
@@ -0,0 +1,13 @@
+# graphql
+
+Welcome to the graphql backend plugin!
+
+_This plugin was created through the Backstage CLI_
+
+## Getting started
+
+Your plugin has been added to the example app in this repository, meaning you'll be able to access it by running `yarn start` in the root directory, and then navigating to [/graphql](http://localhost:3000/graphql).
```

Comment:
Looks like this is copied from a frontend plugin readme. I think we could provide a new template for graphql plugins?